rlleague/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
```

Comment:
This shouldn't be the main for the whole project since its one of two services that will be ran.

Reply:
We will reorganize a bit more but I think this might be fine for the moment.